packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -111,6 +128,62 @@ describe('internal-plugin-metrics', () => {
       assert.deepEqual(cdl.getMeetingInfoReqResp(), 10);
     });
 
+    describe('measureLatency', () => {
+      let clock;
+      let callbackStub;
```

Comment:
nitpick: This probably doesn't need to be a top level `let` since it is explicitly defined in each test.